src/attachment.js

```diff
@@ -0,0 +1,48 @@
+export class Attachment {
+  constructor(basicAttachment, roomId, instance) {
+    this.type = basicAttachment.type
+    this.name = basicAttachment.name
+    this.size = basicAttachment.size
+
+    if (basicAttachment.customData) {
```

Comment:
Needs is defined? Is `false` a valid JSON object?

Reply:
Very good point... yes, as is `null` 🤦‍♂️